src/fortranapex/fortranapex.pyf

```diff
@@ -166,65 +57,65 @@ python module fortranapex ! in
             real(kind=8), parameter,optional :: req=6378.1370d0
             real(kind=8), parameter,optional :: eps=1.d0/298.257223563d0
             real(kind=8), parameter,optional,depend(req,eps) :: re=req*(1-eps/3d0)
-            real(kind=8), parameter,optional,depend(eps) :: ecc2=0.0066943799901413165
-            real(kind=4), parameter,optional :: missing=-9999.0
+            real(kind=8), parameter,optional,depend(eps) :: ecc2=-8
```

Comment:
Seems like this wasn't handled correctly, probably need to fix in the fortran.

Reply:
I've made the corrections above and recreated the signature file.  This is the segment now:
```
            real(kind=8), parameter,optional :: req=6378.1370d0
            real(kind=8), parameter,optional :: eps=1.d0/298.257223563d0
            real(kind=8), parameter,optional,depend(req,eps) :: re=req*(1.d0-eps/3.d0)
            real(kind=8), parameter,optional,depend(eps) :: ecc2=eps*(2.d0-eps)
            real(kind=8), parameter,optional :: missing=-9999.0
```